ieeeuoft/newhacks-site-2020
# Add new fields to application model

Resolves #28 

## Changes
Does some formatting to the application page as well to group things together

## Steps to QA
- Set `REGISTRATION_OPEN=True` at the bottom of `hackathon_site/settings/__init__.py`.
- Make yourself an account if you don't already have one, and go to `/registration/application/`
- Look with your eyes at the form, try filling it out as well. You can see your results in the admin site.
